sdk/src/types/block/payload/milestone/option/parameters.rs

```diff
@@ -99,10 +99,10 @@ pub mod dto {
         }
     }
 
-    impl TryFrom<&ParametersMilestoneOptionDto> for ParametersMilestoneOption {
+    impl TryFrom<ParametersMilestoneOptionDto> for ParametersMilestoneOption {
         type Error = Error;
 
-        fn try_from(value: &ParametersMilestoneOptionDto) -> Result<Self, Self::Error> {
+        fn try_from(value: ParametersMilestoneOptionDto) -> Result<Self, Self::Error> {
```

Comment:
Do we want to change it when it's not needed?

Reply:
What do you mean?

Reply:
Uniformity?

Reply:
In other places we change something to prevent a clone, but whatever